src/main/java/com/project/ita5/answer/validator/AnswerValidator.java

```diff
@@ -0,0 +1,55 @@
+package com.project.ita5.answer.validator;
+
+
+import com.project.ita5.answer.Answer;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.Arrays;
+
+public class AnswerValidator implements ConstraintValidator<Conditional, Answer> {
+//    private String[] selected;
+//    private int max;
+    private String message;
+//    private String[] values;
```

Comment:
Commented out code should not be pushed to the repo.

Reply:
Done